src/Stack/Docker.hs

```diff
@@ -244,7 +244,7 @@ runContainerAndExit = do
          liftIO
              (Files.fileExist
                  (toFilePathNoTrailingSep (sandboxHomeDir </> sshRelDir)))
-     when (sshDirExists && not sshSandboxDirExists)
+     when (sshDirExists && not sshSandboxDirExists && not osIsWindows)
```

Comment:
I will check if this is the right thing to do and if yes then provide justification.

Reply:
This has to stay this way at least for now, because unix-compat does not expose Windows' ability to create symbolic links. I will leave it like that for now.